docs/static-run.md

```diff
@@ -1,32 +1,30 @@
 ### Static run
 
-#### Take care : 
+#### Running the testsuite without sumaform
  
-if you want to run the testsuite without sumaform, you have to create/configure the machine manually, or in alternative
-using the salt-states.
+If you want to run the testsuite without sumaform, you have to create and configure the machine manually, or with the help of Salt states.
 
-Best solution is to use sumaform, so every time you have a clean-fresh testing env.
+Best solution is to use sumaform, so every time you have a clean and fresh testing environment.
+With a static setup, if you break your testing machines, or if you have any trouble, you are on your own.
 
-For the static setup, if you break your testing-machines, you have any troubles, you are on your own, with the static setup.
+Set up the following environment variables:
 
-Once you have the machines configured, you can run the rake and tests.
+* TESTHOST the SUSE Manager server you are testing against
+* CLIENT the traditional client
+* MINION the Salt minion
+* SSHMINION the SSH-managed Salt minion
+* CENTOSMINION the CentOS Salt minion
+* BROWSER (default `phantomjs`) the browser: `chrome`, `htmlunit`, `chrome`, `firefox`
```

Comment:
This variable `Browser` doesnt exist anymore, and it will not exist in longterm. ( we don't have the capacity to support /debug multibrowsers) . you can remove it